Python/Pdf2Video_Converter/Readme.md

```diff
@@ -0,0 +1,33 @@
+# Pdf2Video Converter
+## Short description
+It will take a pdf file and a video as input and Add text audio of pdf to the Given Video.
+## Setup instructions
+In order to run this script, you need to have Python and pip installed on your system. After you're done installing Python and pip, run the following command from your terminal to install the requirements from the same folder (directory) of the project.
+```
+pip3 install -r requirements.txt
+```
+
+After satisfying all the requirements for the project:-
+
+- Place the required pdf and video in same folder as of script.
+- Run following command :-
+    ```
+    python script.py
+    ```
+    or
+    ```
+    python3 script.py
+    ```
+Depending upon the python version. Make sure that you are running the command from the same virtual environment in which the required modules are installed.
+
+
+## Output
+User is asked for a Pdf file and video(mp4) file and Output Video file is stored in the folder.
+### Sample Output
+
+
+https://user-images.githubusercontent.com/55010599/124347215-0c0a1f00-dc01-11eb-9e19-7b36cf78557f.mp4
```

Comment:
Use imgur.com as Content delivery network as it is already mentioned in contributing guidelines